src/maks.kulykov/DatabaseManager.java

```diff
@@ -0,0 +1,48 @@
+package maks.kulykov;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private String db = "mydb";
+    private String userName = "postgres";
+    private String password = "postgres";
+
+    private String cmdDB;
+    private String cmdUserName;
+    private String cmdPassword;
+
+    public DatabaseManager(String db, String userName, String password) {
```

Comment:
I'd add user's input as parameters in 'checkCredentials()'. Then you don't need to save them as class fields.

Reply:
+